index.html

```diff
@@ -405,48 +405,27 @@ <h3>Terminology and Notation</h3>
+            <p class=syntaxExample><span class=lnum>9 </span>&lt;/<span class="vocabulary">body</span>&gt;</p>
+            <p class=syntaxExample><span class=lnum>10 </span>&lt;/<span class="vocabulary">html</span>&gt;</span> </p>
           </div>
           <ul style="text-align:left">
-          <li>Everything inside the black rectangle (that is, in this HTML file)
-            is part of the resource.</li>
-          <li><a>Syntactic content</a> is shown in a <span class="markup">monospaced font</span>.</li>
-          <li><a>Natural language content</a> is shown in a <span class="shakespeare">bold
-              blue font with a gray background</span>.</li>
-          <li>User values are shown in <span class="userValue">italics</span>.</li>
-          <li><a>Vocabulary</a> is shown with <span class="vocabulary">red underlining</span>.</li>
-          <li>All of the text above (all text in a text file) makes up a
-            resource. It's possible that a given resource will contain no
-            natural language content at all (consider an HTML document
-            consisting of four empty <code>div</code> elements styled to be
-            orange rectangles). It's also possible that a resource will contain
-            <em>no</em> syntactic content and consist solely of natural language content:
-            for example, a plain text file with a soliloquy from <cite>Hamlet</cite>
-            in it. Notice too that the HTML entity <code>&amp;#x2019;</code>
-            appears in the natural language content and belongs to both the
-          natural language content and the syntactic content in this resource.</li>
+          <li>Everything inside the black rectangle (that is, in this HTML file) is part of the <a>resource</a>.</li>
+          <li><a>Syntactic content</a> in this case includes all of the HTML markup. There are only two strings that are <strong>not</strong> part of the syntactic content: the word <em>"Shakespeare"</em> on line 4 and the sentence <em>"What&#x2019;s in a name? That which we call a rose by any other name would smell as sweet."</em> on line 8. (The HTML entity <q><kbd>&amp;#x2019;</kbd></q> embedded in the sentence on line 8 <em>is</em> part of the syntactic content.)</li>
+          <li><a>Natural language content</a> is shown in a <span class="shakespeare">bold blue font with a gray background</span>. In addition to the non-syntactic content, the <kbd>alt</kbd> value on line 7 (<em><q>William Shakespeare</q></em>) contains natural language text.</li>
+          <li>User values are shown in <span class="userValue">italics</span>. In this case there are three user values on line 7: the values of the <kbd>src</kbd>, <kbd>alt</kbd>, and <kbd>id</kbd> attributes of the <kbd>img</kbd> tag. In addition, the value of the <kbd>lang</kbd> attribute on line 1 and the <kbd>charset</kbd> attribute on line 3 are user values.</li>
```

Comment:
Maybe "User values" -> "User-supplied values"

Reply:
We could change this. I'll admit that 'user values' isn't super clear. Want to file an issue?

Reply:
Done.